masonry/src/testing/helper_widgets.rs

```diff
@@ -554,7 +556,17 @@ impl Widget for ReplaceChild {
     }
 
     fn children_ids(&self) -> SmallVec<[WidgetId; 16]> {
-        todo!()
+        smallvec![self.child.id()]
+    }
+
+    // FIXME - We pretend this widget is called SizedBox to avoid modifying every single
+    // snapshot test. We should remove this in a dedicated PR.
```

Comment:
Or just have a dedicated PR for this change as a whole and do it well in one shot.